### DISCUSSION TITLE
Advances of direct maximization of the likelihood over Baum-Welch algorithm

### DISCUSSION THREAD

**Discussion by NikosMemmos** [TheoMichelot/moveHMM#40] [General]
Hi All,

I was wondering why you chose the direct maximization of the likelihood for estimating the parameters over the Baum-Welch algorithm, which has been the gold standard. I am new in the field and I would like to know more about it. Thank you very much!

**Comment by TheoMichelot** [comment]
Hi, I don't have much experience with the Baum-Welch algorithm, but here are some references you might be interested in:

- MacDonald (2014). Numerical maximisation of likelihood: A neglected alternative to EM?. International Statistical Review, 82(2), 296-308.
- Zucchini, MacDonald, & Langrock (2017). Hidden Markov models for time series: an introduction using R. CRC press. _(In particular, Section 4.4 contrasts the Baum-Welch algorithm to direct numerical optimisation.)_

My understanding is that the Baum-Welch algorithm is often more challenging to implement, and some authors have found it slower to find the maximum likelihood estimates, with little benefits over direct optimisation. That being said, I'm sure there are cases where Baum-Welch is better (it might sometimes be more numerically robust, for example).
